tests/vppoutputencode.h

```diff
@@ -50,6 +50,9 @@ class EncodeParams {
     int8_t diffQPIB;// B frame qp minus initQP
     uint32_t temporalLayerNum; // svc-t temporal layer number
     uint32_t priorityId; // h264 priority_id in prefix nal unit
+    //h264 simucast means independent bitstreams with different priority.
+    //It require to set priority_id and prefix nal unit for each bitstream.
+    bool enableSimucast;
```

Comment:
Can we add an overall explanation on how enableSimulcast, priorityID and temporalLayerNum work together?